desktop/app/window-handlers/external-links/index.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * External Dependencies
+ */
+const { URL } = require( 'url' );
+
+/**
+ * Internal dependencies
+ */
+const Config = require( '../../lib/config' );
+const openInBrowser = require( './open-in-browser' );
+const log = require( '../../lib/logger' )( 'desktop:external-links' );
+
+const isWordPress = ( url ) => url.hostname.includes( 'wordpress.com' );
```

Comment:
I wonder if we should be pickier about URLs here when using with `openInBrowser`. 
There can be cases where you could visit a URL like a person's site that uses the WordPress.com subdomain and it breaks things. Especially if the WordPress.com subdomain redirects to a different address or an Atomic site. 
Should we try to gather an allowable list of WordPress.com subdomains to allow but not allow customer domains?

Reply:
@sandymcfadden what breakages were you noticing? We determined that a wpcom site is fine. The redirection will cause Electron to test the new URL and make a determination if it should open in the browser or in app.

Reply:
I know I'm late to this now but just wanted to follow up with what I meant. 
Here is a screen recording of clicking on a wordpress.com subdomain link that is for an Atomic site. You end up in a state where you don't have the admin bar at the top. The same could happen if someone had a Site Redirect and redirected their wordpress.com address to a site hosted elsewhere. 

https://d.pr/i/bfhxI8

Reply:
This is very useful info, @sandymcfadden - thanks for catching this. I think I get the gist of the issue, but it would help if you laid out the steps to reproduce in a little bit more detail. How does one go about creating a test site that will lead to this issue? Will any Atomic site have this problem?